### PR TITLE
Rename "Game engines" to "Game Development"

### PR DESCRIPTION
**Pull request by kzys**
See #1598 for the discussion.

### CODE REVIEW

**Comment by kzys** [comment]
Ah, changing the slug means that we have to make some changes on the production database as well...

**Comment by kzys** [comment]
So, we cannot merge the change as is. Renaming "Game Engines" to "Game Development" makes sense though.

**Comment by jtgeibel** [comment]
Would it be better to just add a new slug?  I think it may be confusing if developers still have to add the "game-engine" slug to their `Cargo.toml` even though the category has been renamed.

**Comment by sgrif** [comment]
I think it'd be even more confusing to have a separate "Game Development" category ("Game Engines" aren't for "Game Development"?). We definitely can't just change the slug though, even if we updated the existing data we'd unlist all existing crates when they published a new version.

We should probably have some form of migration strategy here rather than just assuming we'll always get categories exactly right the first time. We'll likely need to add some form of `aliases` field to categories and use that for lookup as well on publish. I agree with @jtgeibel that it'd be confusing to change the display name by itself.

File: src/boot/categories.toml [hunk 1]
```diff
@@ -225,7 +225,7 @@ Crates for dealing with files and filesystems.\
 """
 
 [game-engines]
-name = "Game engines"
+name = "Game Development"
```
Comment (sgrif) [inline src/boot/categories.toml]: None of the other categories capitalize words other than the first.

**Comment by kzys** [comment]
Sorry, I wasn't aware the impact of this change.

Game Engines are for Game Development, but the description ("Crates for creating games.") and the existing creates are more aligned with "Game Development". Game engines usually refer full-fledged frameworks such as Unity and/or Unreal Engine.

We already have a few categories that are the subset of the existing categories, such as Algorithms vs. Compression. I think having separate "Game Engine" is okay, while we may ask the Rust community to migrate their crates from Game Engine to Game Development though.

Having aliases doesn't need this community effort, but adds some complexity on the crates.io.

**Comment by carols10cents** [comment]
I'm sorry we've let this sit so long without feedback!

I think adding a new "Game development" category with a description that it's for not-engines and adjusting the description for "Game engines" so that it's clear what belongs there vs. "Game development" would be a good path forward.

**Comment by kzys** [comment]
Thanks. Will update the TOML soon.

**Comment by carols10cents** [comment]
Noticed this PR was updated; looks great now! Thank you!!

@bors r+

**Pull request merged**